src/main/target/NOX/target.h

```diff
@@ -0,0 +1,118 @@
+/*
+ * This is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this software.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define TARGET_CONFIG
+
+#define TARGET_BOARD_IDENTIFIER "NOX1"
+#define USBD_PRODUCT_STRING "NoxF4V1"
+
+#define LED0_PIN                PA4
+
+#define BEEPER                  PC13
+#define BEEPER_INVERTED
+
+#define INVERTER_PIN_UART2      PC14
+
+#define ACC
```

Comment:
These won't work in 3.3, it's `USE_<x>` now.

Reply:
Is `USE_BEEPER` valid?

Reply:
Nope. `BEEPER` is a pin definition, so it should probably be changed to `BEEPER_PIN`, and potentially a `USE_BEEPER`  added. Thoughts, @martinbudden?

Reply:
Agreed

Reply:
Got it.

Reply:
(And I did not mean to say that this should be part of this pull request, just leave it at `BEEPER` here, @jflyper. But now would be a good time to do a pull request to split `BEEPER` into `USE_BEEPER` and `BEEPER_PIN`.)

Reply:
Yeah, let's do that first.